plugins/out_stackdriver/stackdriver.c

```diff
@@ -2567,13 +2567,14 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
     else {
         flb_http_add_header(c, "User-Agent", 10, "Fluent-Bit", 10);
     }
-
-    flb_http_add_header(c, "Content-Type", 12, "application/json", 16);
-    flb_http_add_header(c, "Authorization", 13, token, flb_sds_len(token));
     /* Content Encoding: gzip */
     if (compressed == FLB_TRUE) {
+        flb_http_add_header(c, "Content-Type", 12, "text/plain", 10);
```

Comment:
The `Content-Type: text/plain` on that page is just an example. In this case, the content type is `application/json`, so that's what you should set it to (with `Content-Encoding: gzip`). Setting it to `text/plain` is wrong. Let's instead figure out why `application/json` didn't work.

Reply:
If it's wrong, what will happen? It won't be json in cloud logging?

I can check cloud logging UI.

Reply:
Verified, json is working